src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/DataSetsControllerTests.cs

```diff
@@ -517,4 +517,171 @@ public async Task InvalidDataSetId_Returns404()
             return await client.GetAsync(uri);
         }
     }
+
+    public class GetVersionTests : DataSetsControllerTests
+    {
+        public GetVersionTests(TestApplicationFactory testApp) : base(testApp)
+        {
+        }
+
+        [Theory]
+        [InlineData(DataSetVersionStatus.Published)]
+        [InlineData(DataSetVersionStatus.Unpublished)]
+        [InlineData(DataSetVersionStatus.Deprecated)]
+        public async Task VersionIsAvailable_Returns200(DataSetVersionStatus dataSetVersionStatus)
+        {
+            DataSet dataSet = DataFixture
+                .DefaultDataSet()
+                .WithStatusPublished();
+
+            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSets.Add(dataSet));
+
+            DataSetVersion dataSetVersion = DataFixture
+                .DefaultDataSetVersion(
+                    filters: 1,
+                    indicators: 1,
+                    locations: 1,
+                    timePeriods: 3)
+                .WithStatus(dataSetVersionStatus)
+                .WithPublished(DateTimeOffset.UtcNow)
+                .WithDataSetId(dataSet.Id);
+
+            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSetVersions.Add(dataSetVersion));
+
+            var response = await GetVersion(dataSet.Id, dataSetVersion.Version);
```

Comment:
Not a necessity, but might be worth using named parameters. Similar could apply in any of the new tests too

Reply:
I think we agreed to use them when we had 3 or more parameters (under any circumstances), and always use them when we have 2 or more parameters of the same type?

I'm happy to adjust this to use named parameters but just want to be consistent with the coding standards so we're consistent across the codebase

Reply:
Yeh I don't think it's a necessity. As you've said, it's not breaking any style guide rules. 

I'd personally have written them as named parameters just for better readability, but happy to leave it if you'd prefer it as-is!